doc/riscv-bfloat16-format.adoc

```diff
@@ -0,0 +1,73 @@
+[[bfloat16_format]]
+== BFloat16 Operand Format
+
+BFloat16 bits::
+[wavedrom, , svg]
+....
+{reg:[
+{bits: 1, name: 'S'},
+{bits: 5, name: 'expo'},
+{bits: 10, name: 'frac'},
+]}
+....
+
+IEEE Compliance: While BFloat16 (BF16) is not an IEEE-754 _standard_ format, it is a valid floating point format as defined by the standard. There are three parameters that specify a format: radix (b), number of digits in the significand (p), and maximum exponent (emax).
+For BF16 these values are:
+
+[%autowidth]
+.BFloat16 paramenters
+|===
+|radix (b)|2
+|significand (p)|8
+|emax|127
+|===
+
+
+.Obligatory Floating Point Format Table
+[cols = "1,1,1,1,1,1,1,1"]
+|===
+|Format|Sign Bits|Expo Bits|fraction bits|padded 0s|encoding bits|expo max/bias|expo min
+
+|FP16    |1| 8| 7| 0|16| 127|-126
+|BFloat16|1| 5|10| 0|16|  15| -14

```

Comment:
I think you got these switched around: BF16 is the same as FP32, but with 16 fewer fraction bits: https://cloud.google.com/blog/products/ai-machine-learning/bfloat16-the-secret-to-high-performance-on-cloud-tpus

Reply:
You are correct. Somehow I swapped them.